gallery/psget/module/RequireLicenseAcceptance.md

```diff
@@ -0,0 +1,213 @@
+---
+ms.date:  2017-06-09
+schema:  2.0.0
+locale:  en-us
+keywords:  powershell
+online version:  
+external help file:  
```

Comment:
Remove these two columns too. these are not required here as this is not a cmdlet reference doc,

Reply:
Done

---
In reply to: [136226307](https://github.com/PowerShell/PowerShell-Docs/pull/1620#discussion_r136226307) [](ancestors = 136226307)